tools/types/src/server.ts

```diff
@@ -55,13 +49,10 @@ export type IndexedOrder<Type> = {
 export type OrderResponse<Type> = {
   orders: Record<string, IndexedOrder<Type>>
   pagination: Pagination
-  filters?: FiltersResponse | undefined
-  ordersForQuery: number
 }
 
 export type Pagination = {
-  first: string
-  last: string
-  prev?: string | undefined
-  next?: string | undefined
+  limit: number
+  offset: number
+  totalForQuery: number
```

Comment:
```suggestion
  total: number
```

I think Don said `total`. 😁